cve_bin_tool/checkers/hunspell.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for hunspell
+
+https://www.cvedetails.com/product/60308/?q=Hunspell
+
+"""
+from cve_bin_tool.checkers import Checker
+
+
+class HunspellChecker(Checker):
+    CONTAINS_PATTERNS = [
+        r"Example: hunspell -d en_US file.txt    # interactive spelling"
+    ]
+    FILENAME_PATTERNS = [r"hostapd"]
```

Comment:
This seems unlikely to be correct, and if it is, it's going to cause false positives with https://en.wikipedia.org/wiki/Hostapd

Reply:
that happened since I used hostapd checker file as boilerplate and somehow `FILENAME_PATTERNS` didn't get caught in my eyes. Now I wonder if long tests doesn't depend on `FILENAME_PATTERNS`

Reply:
I believe if you have duplicate filename patterns (and duplicate version matching patterns) cve-bin-tool will match both, so I wouldn't expect either test to fail.  We had an issue suggesting we should make some match-this-but-any-other-known-checker tests but they haven't been implemented.  It'd be pretty simple to write an assert that does that, although you'd have to allow for some to double-match with glibc I imagine.